koku/koku/middleware.py

```diff
@@ -61,7 +75,13 @@ class HttpResponseUnauthorizedRequest(HttpResponse):
     Used if identity header is not sent.
     """
 
-    status_code = 401
+    status_code = HTTPStatus.UNAUTHORIZED
+
+
+class HttpResponseFailedDependency(HttpResponse):
+    """A subclass of HttpResponse to return a 424."""
+
+    status_code = HTTPStatus.FAILED_DEPENDENCY
```

Comment:
Should we add a JSON error payload?  That would allow us to indicate that it was Postgres or RBAC (next PR) failing.  It could follow similar formatting to other error JSON bodies we provide (despite being 424).

Reply:
You may not need to modify this class at all. The super constructor will do that. See https://github.com/project-koku/koku/blob/master/koku/api/sources/view.py#L89-L93

Reply:
looks good